packages/concerto-tools/test/codegen/fromOpenApi/cto/data/smallStripeOpenApiDefinition.json

```diff
@@ -0,0 +1,659 @@
+                      "format": "unix-time",
+                      "type": "integer"
+                  },
+                  "id": {
+                      "description": "Unique identifier for the object.",
+                      "maxLength": 5000,
+                      "type": "string"
+                  },
+                  "metadata": {
+                      "additionalProperties": {
+                          "maxLength": 500,
+                          "type": "string"
+                      },
+                      "description": "Set of [key-value pairs](https://stripe.com/docs/api/metadata) that you can attach to an object. This can be useful for storing additional information about the object in a structured format.",
+                      "type": "object"
+                  }
+              },
+              "required": [
+                  "created",
+                  "id"
+              ],
+              "title": "Customer",
+              "type": "object",
+              "x-resourceId": "customer"
+          }
+      }
+    },
+    "openapi": "3.0.0",
+    "paths": {
+      "/v1/customers/{customer}": {
+        "get": {
+          "description": "<p>Retrieves a Customer object.</p>",
+          "operationId": "GetCustomersCustomer",
+          "parameters": [
+            {
+              "in": "path",
+              "name": "customer",
+              "required": true,
+              "schema": {
+                "maxLength": 5000,
+                "type": "string"
+              },
+              "style": "simple"
+            },
+            {
+              "description": "Specifies which fields in the response should be expanded.",
+              "explode": true,
+              "in": "query",
+              "name": "expand",
+              "required": false,
+              "schema": {
+                "items": {
+                  "maxLength": 5000,
+                  "type": "string"
+                },
+                "type": "array"
+              },
+              "style": "deepObject"
+            }
+          ],
+          "requestBody": {
+            "content": {
+              "application/x-www-form-urlencoded": {
+                "encoding": {
+                },
+                "schema": {
+                  "additionalProperties": false,
+                  "properties": {
+                  },
+                  "type": "object"
+                }
+              }
+            },
+            "required": false
+          },
+          "responses": {
+            "200": {
+              "content": {
+                "application/json": {
+                  "schema": {
+                    "anyOf": [
+                      {
+                        "$ref": "#/components/schemas/customer"
+                      },
+                      {
+                        "$ref": "#/components/schemas/deleted_customer"
```

Comment:
There are validation errors in this OpenAPI spec. Is that deliberate?

Reply:
Addressed ✅